File: API/Entities/BarberService.cs
namespace API.Entities
{
    public class BarberService
    {
        public int Id { get; set; }
        public int BarberId { get; set; }
        public Barber Barber { get; set; }
        public int ServiceId { get; set; }
        public Service Service { get; set; }
        public double Price { get; set; }
    }
}

File: client/src/app/models/barberService.ts
import { Barber } from './barber';
import { Service } from './service';

export interface BarberServiceModel {
  id: number;
  barberId: number;
  barber: Barber;
  serviceId: number;
  service: Service;
  price: number;
  isNew: boolean;
  availableServices: Service[];
}
